TileStache/Goodies/VecTiles/transform.py

```diff
@@ -685,6 +685,32 @@ def overlap(feature_layers, base_layer, cutting_layer,
         target_attribute, cutting_attrs, keep_geom_type)
 
 
+# map from old or deprecated kind value to the value that we want
+# it to be.
+_deprecated_landuse_kinds = {
+    'station': 'substation',
```

Comment:
Gotcha. It works for today's selections so "works for me". By the time we fix https://github.com/mapzen/vector-datasource/issues/132 and add in railway to the landuse selection we might need to revise.